src/components/ReviewConfirm/Summary.vue

```diff
@@ -1,22 +1,36 @@
 <template>
   <div class="summary-container">
-    <SummaryDefineCompany/>
+    <SummaryDefineCompany />
+    <ListPeopleAndRoles :personList="orgPersonList" :isSummary="true" :showErrorSummary="!stepValid"/>
   </div>
 </template>
 
 <script lang="ts">
 // Libraries
 import { Component, Vue } from 'vue-property-decorator'
+import { State } from 'vuex-class'
 
 // Components
 import { SummaryDefineCompany } from '@/components/Summary'
+import { ListPeopleAndRoles } from '@/components/AddPeopleAndRoles'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
 
 @Component({
   components: {
+    ListPeopleAndRoles,
     SummaryDefineCompany
   }
 })
-export default class Summary extends Vue { }
+export default class Summary extends Vue {
+  // State
+  @State(state => state.stateModel.addPeopleAndRoleStep.orgPeople)
+  readonly orgPersonList: OrgPersonIF[]
+
+  @State(state => state.stateModel.addPeopleAndRoleStep.valid)
+  readonly stepValid: boolean
```

Comment:
I recommend that you rename this, as you may need something similar for steps 3 and 4.

Reply:
Good point, thanks. Done!